components/common/ButtonWithDropdown.vue

```diff
@@ -23,11 +23,14 @@
             </li>
             <li>
              <app-link to="https://github.com/oceanprotocol/oceandao/wiki/Grant-Proposal-Template">
-                <app-button-style
-                  class="px-4 py-3 w-full text-center"
-                  :icon="require('@/assets/images/icons/vote-white.svg')"
-                  :text="$t('project.proposal.submit')"
+              <div class="flex px-4 py-3 text-primary">
+                <img
+                  class="mr-2"
+                  src="@/assets/images/icons/vote.svg"
+                  :alt="$t('project.proposal.submit') + ' ' + $t('general.icon')"
                 />
+                <p>{{$t('project.proposal.submit')}}</p>
```

Comment:
chore(blocking): apply eslint fix. You can also use a plugin for that to automatically fix linting